### ISSUE TITLE
ValuesQuerySet has been removed in django 1.9

### ISSUE THREAD

**Issue by gg-mmill** [sbdchd/django-types#223]
Hello,
trying to type a return of `values_list`, I stumbled upon `ValuesQuerySet`, which is still part of django-types, but has been removed in django 1.9. Is there a reason this is still here ? (I realize this is a complex problem, having to support multiple django versions is probably quite troublesome :/)

**Comment by sbdchd** [comment]
Yeah should be good to remove! I don't think we need to support anything too ancient
